static/css/badge.less

```diff
@@ -35,7 +35,7 @@
   z-index: 100;
   font-size: 0.5rem;
   margin: 0;
-  right: 20px;
+  right: 15%;
```

Comment:
This is quite a change, right, conceptually? I know we're trying to move to rems everywhere for example, but do we have CSS guidelines around things like this?

Reply:
No guidelines that I know of. Essentially we want the badge to appear near the top right of variable with elements. If the element was, say, 60px and the badge was 20px wide then it would appear to be centered. This is a way to ensure that the badge gets closer to the right end of the parent element as it gets squished down on smaller devices.